### DISCUSSION TITLE
Inference for custom trained finetuned models

### DISCUSSION THREAD

**Discussion by goutham-pasumarthy** [huggingface/text-embeddings-inference#79] [Q&A]
Help regarding loading custom trained models

Hi, is it possible to add support for inference of Keras Models?

**Comment by OlivierDehaene** [comment]
As long as you can export weights with the correct layout to Safetensors, yes.

**Reply by goutham-pasumarthy** [reply]
I see, is there any documentation for loading the models from a local directory? Any argument to pass for offline/local models? It is looking for URL when I passed the local directory path to --model-id
